<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="floor1up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 1 Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 2 Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 3 Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2down">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 2 Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3down">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 3 Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor4down">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 4 Down</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floors</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator1f1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator1f2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator1f4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator1f3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2f3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2f1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2f4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2f2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fire_button">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>390</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire Emergency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerout_button">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>430</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Outage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2help">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HELP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator1help">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HELP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator1doorObstacle">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevatorOpen">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevatorClose">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2Open">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2doorObstacle">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevator2Close">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>460</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Overload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Overload</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>350</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Administrator View</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 1: Safety</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 2: Safety</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e1evator1Display">
    <property name="geometry">
     <rect>
      <x>922</x>
      <y>40</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e1evator2Display">
    <property name="geometry">
     <rect>
      <x>1132</x>
      <y>40</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 2 Display</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 1 Display</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="elevator1Weight">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="elevator2Weight">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>420</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1310</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
